client/state/selectors.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import {
+	property,
+} from 'lodash';
+
+export const getProductsListState = property( 'productsList' );
```

Comment:
I believe we were thinking of creating a global selector file, but I'm not sure we should sneak one in this PR. Do you mind keeping this at the `state/products-list/selectors.js` level until we get more momentum on that?

cc @aduth